generator/src/main/java/com/scottlogic/deg/generator/generation/GenerationConfig.java

```diff
@@ -47,6 +49,10 @@ public TreeWalkerType getWalkerType() {
         return this.walkerType;
     }
 
+    public ReductionTarget getReductionTarget(){
```

Comment:
This is specific to the reductive walker, is there a way to remove this from a 'strategy & mode agnostic' configuration